client/components/Ratings.module.css

```diff
@@ -0,0 +1,40 @@
+.ratingsContainer{
+  display: flex;
+  width: 100%;
+  flex-wrap: row wrap;
+  flex-direction: row;
```

Comment:
I think you would only need 'flex-wrap: wrap' and can remove both row and the whole flex-direction to accomplish the same results.